core/tests/protractor/learnerDashboard.js

```diff
@@ -39,210 +42,309 @@ var SubscriptionDashboardPage =
-    general.waitForSystem();
+    collectionEditorPage.searchForAndAddExistingExploration(
+      'Collection Exploration');
     collectionEditorPage.saveDraft();
-    browser.waitForAngular();
-    general.waitForSystem();
-    element(by.css('.protractor-test-commit-message-input')).sendKeys('Update');
+    element(by.css('.protractor-test-commit-message-input')).sendKeys(
+      'Add Collection Exploration.');
     browser.driver.sleep(300);
     collectionEditorPage.closeSaveModal();
     general.waitForSystem();
-    browser.driver.sleep(300);
     users.logout();
 
-    users.login('learner@learnerDashboard.com');
+    // Verify 'Test Collection' is now in the incomplete section.
+    users.login('learner4@learnerDashboard.com');
     learnerDashboardPage.get();
-    browser.waitForAngular();
-    general.waitForSystem();
     learnerDashboardPage.navigateToIncompleteCollectionsSection();
-    learnerDashboardPage.expectTitleOfSummaryTileToMatch('Test Collection');
+    learnerDashboardPage.expectTitleOfCollectionSummaryTileToMatch(
+      'Test Collection');
     users.logout();
   });
 
   it('displays learners subscriptions', function() {
-    users.login('learner@learnerDashboard.com');
+    users.createUser('learner1@learnerDashboard.com',
+      'learner1learnerDashboard');
+    var creator1Id = 'creatorName';
+    users.createUser(creator1Id + '@learnerDashboard.com', creator1Id);
+    var creator2Id = 'collectionAdm';
+    users.createUser(creator2Id + '@learnerDashboard.com',
+      creator2Id);
+    users.login(creator1Id + '@learnerDashboard.com');
+    workflow.createAndPublishExploration(
+      'Activations',
+      'Chemistry',
+      'Learn about different types of chemistry activations.',
+      'English'
+    );
+    users.logout();
 
+    users.login('learner1@learnerDashboard.com');
     // Subscribe to both the creators.
-    browser.get('/profile/creator1learnerDashboard');
-    browser.waitForAngular();
+    subscriptionDashboardPage.navigateToUserSubscriptionPage(creator1Id);
     subscriptionDashboardPage.navigateToSubscriptionButton();
-    browser.get('/profile/creator2learnerDashboard');
-    browser.waitForAngular();
+    subscriptionDashboardPage.navigateToUserSubscriptionPage(creator2Id);
     subscriptionDashboardPage.navigateToSubscriptionButton();
 
+    // Completing exploration '14' to activate /learner_dashboard
+    libraryPage.get();
+    libraryPage.findExploration('Activations');
+    libraryPage.playExploration('Activations');
+    explorationPlayerPage.expectExplorationNameToBe('Activations');
+    explorationPlayerPage.rateExploration(4);
+
     // Both creators should be present in the subscriptions section of the
     // dashboard.
     learnerDashboardPage.get();
-    browser.waitForAngular();
-    general.waitForSystem();
     learnerDashboardPage.navigateToSubscriptionsSection();
-    learnerDashboardPage.expectSubscriptionFirstNameToMatch('creator...');
+    // LIFO.
```

Comment:
Perhaps expand on this comment a bit (I know what you mean but it's a bit cryptic as written).

Reply:
Done!